### ISSUE TITLE
Emoji 1.6.1 does not work on Thunderbird 52.9.1 (1.5.8 does)

### ISSUE THREAD

**Issue by MichaMeyer** [mganss/EmojiAddIn#26]
### Application and version where the issue occurs (e.g. Thunderbird 52.0, Outlook Desktop 2016, Outlook Web)
Thunderbird 52.9.1
### Expected behavior
inserting emoji image into Compose Window by click
### Actual behavior
nothing happens.. nothing is included on click

**Comment by mganss** [comment]
I can only test using the current stable Thunderbird version, Currently this is 60.5.0. Can't you update to the current version?

**Comment by MichaMeyer** [comment]
No, because a lot of Addons I use do not work anymore mith new Thunderbird!
 

Gesendet: Freitag, 08. Februar 2019 um 12:51 Uhr
Von: "Michael Ganss" <notifications@github.com>
An: mganss/EmojiAddIn <EmojiAddIn@noreply.github.com>
Cc: MichaMeyer <micha.meyer@gmx.net>, Author <author@noreply.github.com>
Betreff: Re: [mganss/EmojiAddIn] Emoji 1.6.1 does not work on Thunderbird 52.9.1 (1.5.8 does) (#26)


I can only test using the current stable Thunderbird version, Currently this is 60.5.0. Can't you update to the current version?

**Comment by mganss** [comment]
Can you check if there's an error message in the developer console when you click on an emoji?

**Comment by MichaMeyer** [comment]
Das Problem ist.das ich dann die alte Version wieder deinstallieren muss... Die neue wieder drauf und dann testen... Und dann wieder umgekehrt...

Ich denke eine portable Thunderbird Version zum Test wäre doch für einen Entwickler die beste Lösung?
-- 
Diese Nachricht wurde von meinem Android Mobiltelefon mit GMX Mail gesendet.Am 22.02.19, 11:20, Michael Ganss <notifications@github.com> schrieb:
Can you check if there's an error message in the developer console when you click on an emoji?

—You are receiving this because you authored the thread.Reply to this email directly, view it on GitHub, or mute the thread.
{"api_version":"1.0","publisher":{"api_key":"05dde50f1d1a384dd78767c55493e4bb","name":"GitHub"},"entity":{"external_key":"github/mganss/EmojiAddIn","title":"mganss/EmojiAddIn","subtitle":"GitHub repository","main_image_url":"https://github.githubassets.com/images/email/message_cards/header.png","avatar_image_url":"https://github.githubassets.com/images/email/message_cards/avatar.png","action":{"name":"Open in GitHub","url":"https://github.com/mganss/EmojiAddIn"}},"updates":{"snippets":[{"icon":"PERSON","message":"@mganss in #26: Can you check if there's an error message in the developer console when you click on an emoji?"}],"action":{"name":"View Issue","url":"https://github.com/mganss/EmojiAddIn/issues/26#issuecomment-466348299"}}}
[
{
"@context": "http://schema.org",
"@type": "EmailMessage",
"potentialAction": {
"@type": "ViewAction",
"target": "https://github.com/mganss/EmojiAddIn/issues/26#issuecomment-466348299",
"url": "https://github.com/mganss/EmojiAddIn/issues/26#issuecomment-466348299",
"name": "View Issue"
},
"description": "View this Issue on GitHub",
"publisher": {
"@type": "Organization",
"name": "GitHub",
"url": "https://github.com"
}
}
]

**Comment by mganss** [comment]
I meant test with the old version (52). It works in the new version, so no need to test there.

**Comment by MichaMeyer** [comment]
yes, i know.. I meant that i have to uninstall Emoji 1.5.8 and install Emoji 1.6.1, and after testing return to working Emoji 1.5.8. So then you do any changes and we need to test this again... 

so it would be a good idea for you to use an portable version of TB 52.9.1 to test it.. if it can not be reproduced, maybe I will help... but it's not efficent..

**Comment by mganss** [comment]
I haven't found an older portable version of Thunderbird, only the latest stable. But it seems you can also have different versions of Thunderbird installed [side-by-side](http://kb.mozillazine.org/Go_back_to_an_old_version_of_Thunderbird). I'll try it once I have some time.
